components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/APIKeyValidator.java

```diff
@@ -589,13 +595,14 @@ private APIInfoDTO mapToAPIInfo(ArrayList<URITemplate> uriTemplates, String cont
     }
 
     /**
+     * @param messageContext     The message context
      * @param context     API context of API
      * @param apiVersion  Version of API
      * @param requestPath Incoming request path
      * @param httpMethod  http method of request
      * @return verbInfoDTO which contains throttling tier for given resource and verb+resource key
      */
-    public VerbInfoDTO getVerbInfoDTOFromAPIData(String context, String apiVersion, String requestPath, String httpMethod)
+    public VerbInfoDTO getVerbInfoDTOFromAPIData(MessageContext messageContext, String context, String apiVersion, String requestPath, String httpMethod)
```

Comment:
exceeding the max length

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/9250